src/NormalJob.ts

```diff
@@ -0,0 +1,49 @@
+import { Job } from './Job'
+
+/**
+ * A job for a Daemon to complete.
+ */
+export class NormalJob implements Job {
+	/**
+	 * @param name - The job's name. Must be unique between jobs in the same dependency graph.
+	 * @param prerequisites - An optional array of jobs that must be completed before this job can start.
+	 */
+	constructor(
+		private name: string,
+		private prerequisites: Set<Job> = new Set(),
+	) {}
+
+	public getName(): string {
```

Comment:
I usually like to be explicit about visibility because it 1) forces me to think about it and 2) shows my intent better (I think). But it's nbd so I can remove them if you feel strongly about it.